src/server/report/app.js

```diff
@@ -312,6 +306,7 @@ class App extends LitElement {
 		return renderTabButtons('browser results', tabs, index => {
 			this._selectedBrowserIndex = index;
 			this.shadowRoot.querySelector('.tab-panels').scrollTo(0, 0);
+			this.requestUpdate();
```

Comment:
`_selectedBrowserIndex` isn't a reactive state property anymore since we modify it ourselves. But in this case they've selected a new tab so we force a re-render.